lib/workos/errors.rb

```diff
@@ -27,7 +31,12 @@ def initialize(message: nil, http_status: nil, request_id: nil)
     def to_s
       status_string = @http_status.nil? ? '' : "Status #{@http_status}, "
       id_string = @request_id.nil? ? '' : " - request ID: #{@request_id}"
-      "#{status_string}#{@message}#{id_string}"
+      if @error && @error_description
```

Comment:
It's possible that we could have an `error` without and `error_description`, so we should probably account for that case.